README.md

```diff
@@ -16,8 +16,39 @@ To manually install the plugin:
 5. Restart your Jenkins instance after installation is completed.
 
 ## Usage
+
 You can have a look at our [documentation](https://docs.microsoft.com/en-us/azure/service-fabric/service-fabric-cicd-your-linux-applications-with-jenkins) to understand more about building and deploying Linux Service Fabric applications using Jenkins.
 
+## Pipeline
+
+You can call the function `azureServiceFabricPublish` from your Jenkins Pipeline to deploy your application to
+Service Fabric. You can also use the the Jenkins ***Pipeline Syntax*** to generate the Pipeline snippet from
+the form UI.
+
+* Deploy with Azure resource configuration:
+
+   ```groovy
+   azureServiceFabricPublish azureCredentialsId: 'Jenkins-credentials-ID-for-Azure-service-principal',
```

Comment:
@rapatchi @raunakpandya @ArieShout 

How are we able to connect to a cluster using Service Principles? Are you parsing the endpoint from the return somewhere?

Reply:
The logic used to connect to service fabric cluster was unchanged. I implemented the logic to fetch the service fabric details from Azure, using the service principal, resource group and cluster name provided.